x/feemarket/module.go

```diff
@@ -144,15 +144,14 @@ func (am AppModule) RegisterServices(cfg module.Configurator) {
 }
 
```

Comment:
Add to 5 interface 
```
_ module.HasGenesis         = AppModule{}
_ module.HasServices        = AppModule{}
_ appmodule.HasBeginBlocker = AppModule{}
_ appmodule.HasEndBlocker   = AppModule{}
_ module.HasConsensusVersion = AppModuleBasic{}
```

And we can remove unused fucntion. 
RegisterRESTRoutes, RegisterInvariants, RegisterStoreDecoder, GenerateGenesisState, WeightedOperations

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/3f68bb88b9a78e7d3b3eca175997bb81f29fc03d
https://github.com/xpladev/ethermint/pull/14/commits/0dd3e83b3d83abc20727e63bc0dcb5f5c3703dde